client/blocks/reader-post-card/index.jsx

```diff
@@ -2,10 +2,11 @@
  * External Dependencies
  */
 import React, { PropTypes } from 'react';
-import { noop, truncate } from 'lodash';
+import { noop, throttle, truncate } from 'lodash';
 import classnames from 'classnames';
 import ReactDom from 'react-dom';
 import closest from 'component-closest';
+import userModule from 'lib/user';
```

Comment:
I'd like to avoid depending on lib/user.We should either bind Trackable to the redux user, or even better, off load reporting the pixel to the analytics middleware, which can pick up the user info transparently to this component. happy to help with that.